Export/Scripts/pantheons.lua

```diff
@@ -0,0 +1,62 @@
+--
+-- export The Pantheon data
+--
+
+local function zip(a, b)
+    local zipped = { }
+	for i, _ in pairs(a) do
+		table.insert(zipped, { a[i], b[i] })
+    end
+    return zipped
+end
+
+if not loadStatFile then
+	dofile("statdesc.lua")
+end
+loadStatFile("stat_descriptions.txt")
+
+local out = io.open("../Data/3_0/Pantheons.lua", "w")
+out:write('-- This file is automatically generated, do not edit!\n')
+out:write('-- The Pantheon data (c) Grinding Gear Games\n\n')
+out:write('return {\n')
+
+for i, p in pairs(dat"PantheonPanelLayout":GetRowList("IsDisabled", false)) do
+    out:write('\t["', p.Id, '"] = {\n')
+    out:write('\t\tisMajorGod = ', tostring(p.IsMajorGod), ',\n')
+    out:write('\t\tsouls = {\n')
+
+    local gods = {
+        { name = p.GodName1, statKeys = p.Effect1StatsKey, values = p.Effect1Values },
+        { name = p.GodName2, statKeys = p.Effect2StatsKey, values = p.Effect2Values },
+        { name = p.GodName3, statKeys = p.Effect3StatsKey, values = p.Effect3Values },
+        { name = p.GodName4, statKeys = p.Effect4StatsKey, values = p.Effect4Values },
+    }
+    for i, god in pairs(gods) do
```

Comment:
Please change `i`, `j` to something like `god_index`, `soul_index`. Besides, the ijkl notation is most often used for maxtrix operations, so I was a litte confused to see that the inner loop doesn't make use of `i` at all.